sdks/python/apache_beam/utils/pipeline_options_test.py

```diff
@@ -170,6 +172,26 @@ def test_template_location(self):
     options = PipelineOptions(flags=[''])
     self.assertEqual(options.get_all_options()['template_location'], None)
 
+  def test_static_value_provider_of(self):
+    class TestOptions(PipelineOptions):
+      @classmethod
+      def _add_argparse_args(cls, parser):
+        parser.add_argument(
+            '--int_flag',
+            type=static_value_provider_of(int),
```

Comment:
Right, the Java SDK defines options in terms of the ValueProivder interface.  If an option is provided in the arguments, the parser inserts a StaticValueProvider.  If not, the value is a RuntimeValueProvider that must be populated by the service.